source/reporting/index.html.md.erb

```diff
@@ -134,7 +134,7 @@ Example response:
     "three_d_secure": {
       "required": true,
     }
-  }
+  },
```

Comment:
Also looks like a mistake.

Reply:
This one wasn't a mistake - I noticed that a comma was missing after the `refund_summary`. All of the other objects have commas separating them, so I _think_ this one is needed?

Reply:
Oh, yeah, I was wrong; this comma is needed.